docs/source/flow_setup.rst

```diff
@@ -84,66 +89,80 @@ Finally, test your sumo install and version by running the following commands
     which sumo
     sumo --version
     sumo-gui
-    
-If you are a mac user and the above command gives you the error FXApp:openDisplay: unable to open display :0.0 make sure to open the application XQuartz.
 
-3. Flow
-====
-Once sumo and the various dependencies are in place, we are ready to install a
-functional version of Flow. With this, we can begin to simulate traffic within
-sumo using OpenAI gym-compatible environments. Note that separate RL algorithms
-will be needed to train autonomous agents within the simulation to improve
-various traffic flow properties (see the sections on rllab-multiagent and
-Ray/RLlib for more).
-::
+If you are a mac user and the above command gives you the error FXApp:openDisplay: unable to open display :0.0 make sure to open the application XQuartz.
 
-    cd ~
-    git clone https://github.com/flow-project/flow.git
-    cd flow
-    conda env create -f environment.yml
-    source activate flow
-    python setup.py develop
+b. Testing the Installation
+===========================
 
-For linux run
-::
-    echo 'export PYTHONPATH="$HOME/flow:$PYTHONPATH"' >> ~/.bashrc
-    source ~/.bashrc
+Once the above modules have been successfully installed, we can test the
+installation by running a few examples. Before trying to run any examples, be
+sure to enter your conda environment by typing:
 
-For mac run
 ::
-    echo 'export PYTHONPATH="$HOME/flow:$PYTHONPATH"' >> ~/.bash_profile
-    source ~/.bash_profile
 
-4. Testing the Installation
-========================
-
-Once the above modules have been successfully installed, we can test the
-installation by running a few examples.
+    source activate flow
 
 Let’s see some traffic action:
+
 ::
 
     python examples/sumo/sugiyama.py
 
 Running the following should result in the loading of the SUMO GUI.
 Click the run button and you should see unstable traffic form after a
 few seconds, a la (Sugiyama et al, 2008). This means that you have Flow
-properly configured with SUMO.
+properly configured with SUMO and Flow!
+
+c. Easy Install SUMO (optional)
+===============================
+
+In this section, we present and faster and simpler method of installing the
+necessary SUMO binaries and python tools. These setup instructions are still
+experimental, so any and all feedback is greatly appreciated!
+
+In order to install everything you will need from SUMO, run one of the below
+scripts from the Flow main directory. Choose the script that matches the
+operating system you are running.
+
+For Ubuntu 14.04:
+
+::
+
+    scripts/setup_sumo_ubuntu1404.sh
```

Comment:
this is a shell script, do they need to chmod it first?

Reply:
haven't had to so far. this might be because we start the files with
```
#bin/bash
```

Reply:
ah nice